lib/token-types/bearer-token-type.js

```diff
@@ -0,0 +1,52 @@
+
+/**
+ * Module dependencies.
+ */
+
+var InvalidArgumentError = require('../errors/invalid-argument-error');
+
+/**
+ * Constructor.
+ */
+
+function BearerTokenType(accessToken, accessTokenLifetime, refreshToken, scope) {
+  if (!accessToken) {
+    throw new InvalidArgumentError('Missing parameter: `accessToken`');
+  }
+
+  this.accessToken = accessToken;
+  this.accessTokenLifetime = accessTokenLifetime;
+  this.refreshToken = refreshToken;
+  this.scope = scope;
```

Comment:
I think so.  At some point we have to trust users to do the right thing- esp when the spec allows for it.